ansible/zuul.yaml

```diff
@@ -175,6 +175,8 @@
                 - pkg-map
                 - infra-package-needs
                 - zuul-worker
+                - dib-python
```

Comment:
This only installs a symlink during the image build, I don't see where this would be needed?